charts/rancher-monitoring/v0.1.3/questions.yml

```diff
@@ -1 +1 @@
-rancher_min_version: 2.4.6-rc1
+rancher_min_version: v2.5.0-alpha1
```

Comment:
#### Note to Reviewers

@deniseschannon and I chatted offline about this before, we want to bump this `rancher_min_version` until QA validates it onto 2.4; once that is done, this min version will be updated on merge to `release-v2.4`.

cc: @PennyScissors